posthog/management/commands/setup_dev.py

```diff
@@ -16,7 +13,7 @@ def handle(self, *args, **options):
             organization, team, user = User.objects.bootstrap(
                 company_name=ORGANIZATION_NAME,
                 email="test@posthog.com",
-                password="pass",
+                password="12345678",
```

Comment:
Why? Seems unrelated to this PR.

Reply:
Agree that is tangential to the PR, but it's a minor change only intended to aid when running tests locally. This makes sure you can create the test user using the UI too and the password to match.

Reply:
Ack makes sense. This can cause some slight pain (e.g. heroku deploy password has changed) but that makes sense.